src/components/my-app.js

```diff
@@ -36,7 +36,7 @@ import { menuIcon } from './my-icons.js';
 import './snack-bar.js';
 
 class MyApp extends connect(store)(LitElement) {
-  _render({appTitle, _page, _drawerOpened, _snackbarOpened, _offline}) {
+  _render({ appTitle, _page, _drawerOpened, _snackbarOpened, _offline }) {
```

Comment:
We have a todo item to add a formatter to the project. Some discussions [here](https://github.com/Polymer/pwa-starter-kit/pull/154#issuecomment-397412942). Until then I'd suggest trying not to change spaces and such and be more consistent with the other [files](https://github.com/Polymer/pwa-starter-kit/blob/master/src/components/my-view3.js#L37)

FWIW, I do prefer having spaces between curly braces :-)

Reply:
Ack, I forgot to disable prettier. Will undo these changes